.env

```diff
@@ -0,0 +1,2 @@
+SUPABASE_URL=https://gybiadvtikenwgkstajq.supabase.co
+SUPABASE_ANON_KEY=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZSIsInJlZiI6Imd5YmlhZHZ0aWtlbndna3N0YWpxIiwicm9sZSI6ImFub24iLCJpYXQiOjE3MTM4OTI2MDUsImV4cCI6MjAyOTQ2ODYwNX0.6IABMJJT2uv8ITwMyMPm41sA9IBKyo1yy33OiG5wJIY
```

Comment:
Accidentally .env file was committed. I generated a new KEY. This is no longer valid.